src/spikeinterface/postprocessing/tests/common_extension_tests.py

```diff
@@ -76,6 +76,15 @@ def setUp(self):
             overwrite=True,
         )
         self.we2 = we2
+
+        # make we read-only
+        we_ro_folder = cache_folder / "toy_waveforms_2seg_readonly"
+        if not we_ro_folder.is_dir():
+            shutil.copytree(we2.folder, we_ro_folder)
+            # change permissions (R+X)
+        we_ro_folder.chmod(0o555)
```

Comment:
I never learned by memory the chmod changing policy. Maybe I should wirte a post-it in my computer with them until I learn them.

Reply:
I know this is a super future problem, but chmod doesn't fully work with Windows. So you can't change the writeable bit directly using chmod operations (also can't change the executable bit either). If there is ever a common, full testing framework (that Sam approves of) then this may behave unexpected on Windows tests.
https://docs.python.org/3/library/pathlib.html
https://docs.python.org/3/library/os.html#os.chmod

Reply:
Good point! Maybe skipping this if on windows for now! I'll make the suggested changes on monday